server.js

```diff
@@ -16,7 +18,11 @@ server.engine('hbs', hbs({
   defaultLayout: 'main',
   layoutDir: path.join(__dirname, './', 'views/layouts'),
   partialsDir: path.join(__dirname, './', 'views/partials'),
-  extname: 'hbs'
+  extname: 'hbs',
+  helpers: {
+    prettyDate: (date) => dateFormat(date, 'dddd d - mmm - yyyy'),
+    prettyTime: (time) => date.format(new Date(time), 'HH:mm A', true)
```

Comment:
I tried to use dateFormat for a Time but it dosn't  work , and I used another module it date.format